test/functional/RasapiTest/test.xml

```diff
@@ -54,7 +54,7 @@
 				<pathelement location="${ant.home}/lib/ant-launcher.jar"/>
 			</classpath>
 			<jvmarg value="-showversion"/>
-			<jvmarg value="-Xmx128m"/>
+			<jvmarg value="-Xmx512m"/>
```

Comment:
128MB was enough for this test previously - perhaps that's enough for all these tests?

Reply:
My bad, I replaced all 128m with 512m as I was trying to find the minimum value that worked. I've put back this value, and I'll run some new grinders to confirm.

Reply:
The new grinder got OOM on AIX. I've gone back to 512M and re-testing.

Reply:
512M seems to work, when using smaller values did not. I've updated the grinders in the description. It's not worth trying to optimize any further, using 512M is sufficient. What is important is to not use the default -Xmx, which is 25% of physical RAM. The default can produce core files that are too big, taking a long time to write to disk, exhausting disk space, etc.